prisma/schema.prisma

```diff
@@ -62,3 +67,65 @@ model VerificationToken {
 
     @@unique([identifier, token])
 }
+
+model Trip {
+    id                  String      @id @default(cuid())
+    distanceTraveled    Decimal     // user input
```

Comment:
The default unit will be metric system? Or this is up to user to select which measurement unit to use?

Reply:
The default unit field is common to all of the calculator questions and I opted to put it in the User model under 'unitPref'. Otherwise, I've been thinking we should default to metric where applicable since that is what Carbon does on their end.

Reply:
Ah I see. I missed the `unitPref` in the user model. 

And yes, I would default to metric.

Reply:
Resolving this convo. If you're happy with the request, please approve it :)